fppanalysis/two_dim_velocity_estimates.py

```diff
@@ -96,15 +114,16 @@ def __init__(self, ds, estimation_options: EstimationOptions):
         self.z_dim = len(range_z)
         self.ds = ds
         self.estimation_options = estimation_options
-        self.pixels = [[PixelData() for _ in range_z] for _ in range_r]
-
-        from pathos.multiprocessing import ProcessPool
+        self.tde_delegator = tde.TDEDelegator(
+            estimation_options.method,
+            estimation_options.get_time_delay_options(),
+            estimation_options.cache,
+        )
+        self.pixels = [[PixelData() for _ in range_r] for _ in range_z]
 
-        pool = ProcessPool(estimation_options.num_cores)
```

Comment:
Ok, so we completely remove the multiprocessing functionality?

Reply:
Yep